app/views/shared/_header.html.erb

```diff
@@ -1,10 +1,12 @@
 <header>
 
   <a href="/"><img src="/images/logo@2x.png" width="110" height="46" alt="Git" /></a>
-  <span id="tagline">
-    <%= raw random_tagline %>
-  </span>
-
+  <span id="tagline"></span>
+  <script type="text/javascript">
+    var taglines = <%= raw JSON.generate(TAGLINES) %>;
+    var tagline = taglines[Math.floor(Math.random() * taglines.length)];
+    document.getElementById('tagline').innerHTML = '--' + tagline;
+  </script>
```

Comment:
What happens when Javascript is disabled? I think we'd just end up with a blank tagline span. Should we put it in a default static one instead?